hoot-core/src/main/cpp/hoot/core/algorithms/Translator.cpp

```diff
@@ -106,9 +106,7 @@ class JsonDictionary
   }
 };
 
-
-
-Translator* Translator::_theInstance = 0;
+boost::shared_ptr<Translator> Translator::_theInstance = NULL;
```

Comment:
Change the singleton naked pointer to a shared pointer so it isn't leaked.

Reply:
:+1: